### PR TITLE
Optimize sql script use insert ignore

### PR DESCRIPTION
**Pull request by ruanwenjun**
## Purpose of the pull request

- use current_timestamp to replace `2021-xx`
- use insert ignore to avoid duplication exception

### CODE REVIEW

File: dolphinscheduler-dao/src/main/resources/sql/dolphinscheduler_mysql.sql [hunk 1]
```diff
@@ -665,7 +665,7 @@ CREATE TABLE `t_ds_queue` (
 -- ----------------------------
 -- Records of t_ds_queue
 -- ----------------------------
-INSERT INTO `t_ds_queue` VALUES ('1', 'default', 'default', null, null);
+INSERT IGNORE INTO `t_ds_queue` VALUES ('1', 'default', 'default', null, null);
```
Comment (SbloodyS) [inline dolphinscheduler-dao/src/main/resources/sql/dolphinscheduler_mysql.sql:668]: Does postgres need to be modified?

Comment (ruanwenjun) [reply dolphinscheduler-dao/src/main/resources/sql/dolphinscheduler_mysql.sql:668]: It seems PostgreSQL doesn't support insert ignore

Comment (SbloodyS) [reply dolphinscheduler-dao/src/main/resources/sql/dolphinscheduler_mysql.sql:668]: It seems that only versions above 9.5 are supported in postgresql...
`https://stackoverflow.com/questions/1009584/how-to-emulate-insert-ignore-and-on-duplicate-key-update-sql-merge-with-po`

Comment (ruanwenjun) [reply dolphinscheduler-dao/src/main/resources/sql/dolphinscheduler_mysql.sql:668]: Yes, so I don't want to modify the postgresql.

**Review by SbloodyS — APPROVED** [review]
LGTM

**Comment by ruanwenjun** [comment]
This change will not modify the schema.

**Pull request merged**